### PR TITLE
feat: donate with crypto on the bottom right

### LINKED ISSUE #196: New widget > Move “Donate with Crypto, Stocks or DAF” link to the button right
“Donate with Crypto, Stocks or DAF” link to be on the button right corner of the wigdet

Desktop:
image

Mobile:
image

### PR DESCRIPTION
**Pull request by julianpoma**
Closes #196

### CODE REVIEW

File: packages/donate-button-beta/src/components/widget/index.tsx [hunk 1]
```diff
@@ -89,21 +94,29 @@ const nonProfitHeaderCss = cxs({
...
 });
+
 const nonProfitInfoCss = cxs({
 	gridColumn: '2 / 3',
 	gridRow: '2 / 4'
 });
-const ctaCss = cxs({
+
+const cryptoContainerCss = cxs({
+	gridRow: '4 / 4',
```
Comment (martinbianchi) [inline packages/donate-button-beta/src/components/widget/index.tsx:104]: I didn't have the opportunity of look at the result but it seems that here should be `4 / 5`

Comment (julianpoma) [reply packages/donate-button-beta/src/components/widget/index.tsx]: @martinbianchi fixed :) please check

**Comment by martinbianchi** [comment]
The space between [FAQ, Donation Policy] and [Donate with Crypto..] does not seems to be ok, they should be equal.

image

image

**Comment by julianpoma** [comment]
@martinbianchi 

Paddings fixed :D

I also made the frequency popover disappear when you select a freq, take a look

**Review by martinbianchi — APPROVED** [review]

**Pull request merged**